test/e2e/resource_placement_drift_diff_test.go

```diff
@@ -0,0 +1,3045 @@
+/*
+Copyright 2025 The KubeFleet Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"fmt"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/kubefleet-dev/kubefleet/pkg/controllers/workapplier"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/utils/ptr"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	placementv1beta1 "github.com/kubefleet-dev/kubefleet/apis/placement/v1beta1"
+	"github.com/kubefleet-dev/kubefleet/test/e2e/framework"
+)
+
+var _ = Describe("take over existing resources using RP", Label("resourceplacement"), func() {
```

Comment:
So the original design of the drift diff e2e for CRP is that we can test each combination of ApplyStrategy on both `unmanaged` field (tested by the namespace) and `managed` field (tested by the configMap). Right now in our RP test, the unmanaged scenario is stilled tested by CRP but RP only tests the managed field. For each of the test cases, could you please either add another configMap with an unmanaged field change in the placed resources or just add another test case that tests configMap + unmanaged field?